WebApiBackend/WebApiBackend/Controllers/PaymentsController.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using WebApiBackend.Dto;
+using WebApiBackend.EF;
+using WebApiBackend.Interfaces;
+using WebApiBackend.Model;
+
+namespace WebApiBackend.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
Can we try extracting the authorised user to get the user id and flat id, as opposed to allowing any access? I think it'll be much easier to add now, before the frontend depends on these endpoints.

Reply:
We were planning to merge this into master and then make another PR to avoid a larger PR

Reply:
I think it would be better to do this in another pr, this one is already quite large.